### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by Lucas-Erkana**
✅ Modify the Awesome books application to have a Navigation bar with three content sections:
-            - Books list
-            - Add book form
-            - Contact info
✅ Make sure that when you click on any of the links, the main area changes to display only one of the three content sections
✅ Do not use any library or framework, use only plain JavaScript and CSS.
✅ Make Desktop and Mobile versions (632px)
✅ No linter errors
✅ Use the correct GitHub flow
✅ Deploy website

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,73 +1,205 @@
...
+}
 
-// Event: Add a Book
-document.querySelector('#book-form').addEventListener('submit', (e) => {
-  // Prevent actual submit
+document.addEventListener('submit', (e) => {
   e.preventDefault();
+  // if (e.target.title.value === '' || e.target.author.value === '') {
+  //   alert('Input field cannot be empty');
+  // // } else {
```
Comment (fherrerao) [inline main.js]: - [x] Kindly, remember not to leave commented code 🍡, always you should maintain best practices in JavaScript. Please, remove commented code 🔢.

File: main.js [hunk 2]
```diff
@@ -1,73 +1,205 @@
...
-  UI.deleteBook(e.target);
+// Imlplement responsive menu
+
+const menu = document.querySelector('.header');
 
-  // remove book from broswer storage according to author
-  Store.removeBook(e.target.parentElement.previousElementSibling.textContent);
+document.getElementById('humburger').onclick = () => {
+  menu.classList.add('change-menu');
+  // alert('hello')
```
Comment (fherrerao) [inline main.js]: - [x] Kindly, remember not to leave commented code 🍡, always you should maintain best practices in JavaScript. Please, remove commented code 🔢.

**Review by fherrerao — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#
Hi team @Lucash2022, @Kashif-Saleem-Ghuman 💯, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

So Close Snl GIF by Saturday Night Live


## Highlights

✔️  Good use of GitHub flow.
✔️  No linter errors.
✔️  Good README
✔️  Good Commit Messages


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (Lucas-Erkana) [reply main.js]: @fherrerao 
Removed commented code at this [commit](https://github.com/Lucash2022/Awesome_Book/commit/7d237603c0d4e20b0888fbbd42b53f3034bfca1a)

File: main.js (hunk 2 above)
Comment (Lucas-Erkana) [reply main.js]: @fherrerao 
Removed commented code at this [commit](https://github.com/Lucash2022/Awesome_Book/commit/7d237603c0d4e20b0888fbbd42b53f3034bfca1a)

**Review by ichala — APPROVED** [review]
Hi @Lucash2022   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**